### PR TITLE
A few more editorial suggestions

### PR DESCRIPTION
**Pull request by AFBarstow**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -98,7 +114,7 @@
       <p>Here, the <code>PerformanceResourceTiming</code> interface extends the <code>PerformanceEntry</code> interface in the <a href="#performance-timeline">Performance Timeline</a>. The duration attribute in Resource Timing can by measured as the difference between responseEnd and startTime.</p>
       <p>As indicated in <a href="#fig2">Fig.2</a> and <a href="#fig3">Fig.3</a>, you are able to access a set of critical network timing <a href="https://w3c.github.io/resource-timing/#attributes">attributes</a> for each resource on the page.
       </p>
-      <p>Each of these time stamp is in microseconds, as defined by the <code>window.performance.now()</code> method in the<a href="http://www.w3.org/TR/hr-time/#sec-high-resolution-time">High ResolutionTime specification</a>.</p>
+      <p>Each of these time stamp is in microseconds, as defined by the <code>window.performance.now()</code> method in the <a href="http://www.w3.org/TR/hr-time/#sec-high-resolution-time">High ResolutionTime specification</a>.</p>
```
Comment (yoavweiss) [inline index.html:117]: s/time stamp/timestamps/ ?

**Comment by yoavweiss** [comment]
LGTM % nit

**Comment by AFBarstow** [comment]
Re s/time stamp/timestamps/, that WFM, but perhaps it would be useful for a `domain expert` to review this doc.

**Comment by siusin** [comment]
These suggested changes all make sense to me. Much appreciated! @AFBarstow @yoavweiss

**Pull request merged**